### ISSUE TITLE
regarding adaptively modifying the weight in keras training

### ISSUE THREAD

**Issue by wenouyang** [fchollet/keras#5592]
In some paper, I once saw the statement such as "**we train the network for 10 epochs with a learning rate of 0,1m divided by 10 after 5 epochs**" 

How can I adaptively modify the rate in **keras**? In **model.fit**, I cannot find a way to do this.

**Comment by patyork** [comment]
[https://keras.io/callbacks/#learningratescheduler](https://keras.io/callbacks/#learningratescheduler)
